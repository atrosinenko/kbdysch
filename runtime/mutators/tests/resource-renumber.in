[ENV:KBDYSCH_MUTATOR_NUM_SPLICES=0]
[BEGIN:DUMPING OUTPUTS]
[END:CALL DEINIT]
[MUTATE]
%Section 1 %N00
%Section 2 %N01
%Section 3 %N02
%Section 4 %R01 %R02
