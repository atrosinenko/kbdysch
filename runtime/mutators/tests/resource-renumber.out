INJECTOR: Mutator requested 4 mutations.
INJECTOR: OUTPUT #0: BEGIN
INJECTOR: %Section 2 %N01
INJECTOR: %Section 3 %N02
INJECTOR: %Section 4 %R00 %R01
INJECTOR: OUTPUT #0: END
INJECTOR: OUTPUT #1: BEGIN
INJECTOR: %Section 1 %N00
INJECTOR: %Section 3 %N02
INJECTOR: OUTPUT #1: END
INJECTOR: OUTPUT #2: BEGIN
INJECTOR: %Section 1 %N00
INJECTOR: %Section 2 %N01
INJECTOR: OUTPUT #2: END
INJECTOR: OUTPUT #3: BEGIN
INJECTOR: %Section 1 %N00
INJECTOR: %Section 2 %N01
INJECTOR: %Section 3 %N02
INJECTOR: OUTPUT #3: END
